ansible/roles/etcd/tasks/etcd-configure.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Write etcd config file
+  template: src=etcd.conf.j2 dest=/etc/etcd/etcd.conf
+  notify:
+    - restart etcd
```

Comment:
The same here. ``restart etcd`` handler has ``when`` condition with ``etcd_started.changed``. Again, the register variable is not defined at the time of execution of the handler.